src/meta/test/InstallUninstallPluginTest.cpp

```diff
@@ -0,0 +1,146 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
```

Comment:
This file named PluginTest.cpp is better ?

Reply:
good ponit， I will fix it